.travis.yml

```diff
@@ -1,28 +1,34 @@
 language: go
 go:
   - 1.12.4
-os:
-  - linux
-  - osx
-  - windows
 env:
-  - GO111MODULE=on
+  global:
+    - SHA512_CMD=sha512sum
+    - GO111MODULE=on
+matrix:
+  include:
+  - os: linux
+    dist: bionic
+  - os: osx
+    env: SHA512_CMD="shasum -a 512"
+  - os: windows
 addons:
   sonarcloud:
     organization: 030-github
     token:
       secure: $SONAR_TOKEN
 script:
   - diff -u <(echo -n) <(gofmt -d ./)
-  - export DELIVERABLE="golang-bitbucket-cloud-build-status-notifier-${GIMME_OS}"
+  - export DELIVERABLE="golang-bitbucket-cloud-build-status-notifier-${GOOS}"
   - go test -short -cover -v -coverprofile=coverage.out -covermode=atomic ./...
-  - GOARCH=amd64 go build -o $DELIVERABLE
-  - sonar-scanner -Dsonar.projectKey=030_golang-bitbucket-cloud-build-status-notifier -Dsonar.sources=. -Dsonar.host.url=https://sonarcloud.io -Dsonar.coverage.exclusions=main.go,**/*_test.go -Dsonar.go.coverage.reportPaths="coverage.out"
+  - go build -o $DELIVERABLE
+  - $SHA512_CMD $DELIVERABLE > ${DELIVERABLE}.sha512.txt
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then sonar-scanner -Dsonar.projectKey=030_go-yq -Dsonar.sources=. -Dsonar.host.url=https://sonarcloud.io -Dsonar.coverage.exclusions=**/*_test.go -Dsonar.go.coverage.reportPaths="coverage.out"; fi
```

Comment:
go-yq should be changed

Reply:
Ah, missed that one. Can I use any key, or should it be created first?

Reply:
Could you add `"$TRAVIS_PULL_REQUEST" != "false"`  as well? It turned out that encrypted variables are only added for internal code and not for forks because of security reasons